src/templates.c

```diff
@@ -168,7 +171,7 @@ void templates_replace_common(GString *tmpl, const gchar *fname,
 		NULL);
 }
 
-
+// Maybe a fix to do in this function
```

Comment:
I don't know for what this function is useful, but I see it use get_template_fileheader function.
It needs maybe some changes for including gplv3 file header ?

Reply:
It replaces the `{fileheader}` placeholder in other templates when they are loaded.
As noted below, I think we don't need the "fileheader3" variant at all.